src/components/Checkout/BillingInfo/BillingInfo.js

```diff
@@ -16,22 +16,46 @@ class BillingInfo extends React.Component {
   constructor(props) {
     super(props);
 
+    this.state = {
+      inputPassword: '',
+      isChecked: true
+    };
+
     this.form = React.createRef();
     this.validate = this.validate.bind(this);
+    this.setPasswordToState = this.setPasswordToState.bind(this);
+    // this.validatePassword = this.validatePassword.bind(this);
+  }
+
+  setPasswordToState() {
+    const password = this.form.current[21].value;
+
+    this.setState({
+      inputPassword: password
+    });
   }
 
   validate() {
     const { showMessage } = this.props;
     const isInputValid = this.form.current.reportValidity();
+    const { inputPassword } = this.state;
+    const confirmedPassword = this.form.current[22].value;
 
-    if (!isInputValid) {
+    if (!isInputValid || (inputPassword !== confirmedPassword)) {
       showMessage(VALIDATION_FAILED);
     }
 
     return isInputValid;
   }
 
+  handleChange() {
+    // Here will be the proper functionality is added
```

Comment:
If I remove this function there will be a warning in console: 
> checkPropTypes.js:20 Warning: Failed prop type: You provided a `checked` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultChecked`. Otherwise, set either `onChange` or `readOnly`.

I added this "empty" function to remove the warning.

Reply:
`onChange={() => {}}`